capstone/src/main/java/com/google/univiz/api/SearchResource.java

```diff
@@ -11,5 +11,5 @@
    * Take a list of suggestion data and returns a list of Suggestions objects to the caller. This
    * object contains all data relevant to the Search feature.
    */
-  List<SearchData> getSearchSuggestions(List<SuggestionData> collegeNames);
+  List<SearchData> getSearchSuggestions(String collegeNames);
```

Comment:
consider a better more descriptive name for this parameter. Like collegeName, partialCollegeName, collegeNameQuery, searchQuery, etc.

Reply:
updated!